src/http/oauth1.ts

```diff
@@ -125,6 +125,7 @@ export function createOAuth1HttpClient(
         secret: tokenResult.data.tokenSecret,
       });
     }
+    return tokenData;
```

Comment:
it should return the token result, this is just the data that you provide as input, no?

Reply:
if the user provides it, then yes, otherwise it will generate the tokenData and it will return it.

Reply:
There's 2 ways to auth with oauth1 atm. Token and password. If you use token, it's always the same token that's put on to the calls. If you use password, you get new tokens from the /tokens endpoint. Unless you have 2FA enabled. Then you need to use `confirmMfa` to get tokens. So we should return those there as well?